reasoner/common/Tracer.java

```diff
@@ -149,7 +150,7 @@ private void start() {
             try {
                 LOG.debug("Writing reasoner traces to {}", path.get().toAbsolutePath());
                 File file = path.get().toFile();
-                if (!file.getParentFile().mkdirs()) {
+                if (!file.getParentFile().exists() && !file.getParentFile().mkdirs()) {
```

Comment:
mkdirs succeeds iff the directory was created (hence, fails if it already exists).
Adding a .exists() handles this.